llvm/test/Transforms/InstCombine/bitcast-bfloat-half-mixing.ll

```diff
@@ -0,0 +1,70 @@
+; RUN: opt -S %s | FileCheck %s
+
+define double @F0([2 x bfloat] %P0) {
```

Comment:
Are the arrays relevant to this test case, or could we use a scalar instead?

Reply:
We can use scalars